test/utilitiesTest.js

```diff
@@ -51,4 +87,165 @@ contract("Utilities Tests", function(accounts) {
       await testGetMax();
     }
   });
+
```

Comment:
Those test are commented out because they currently rely on hard coded assertions, which usually don't really mean that we are getting the result we want.  It would need to be refactored, in the style of the next two